src/App.js

```diff
@@ -33,21 +34,23 @@ class App extends Component {
   render () {
     console.log(this.state)
     return (
-      <div className="app">
+      <div className="App">
         <Route exact path="/">
           <h1>CarryOkay</h1>
           <p>Hello friend 😬</p>
           <Navigation class="home-nav" />
         </Route>
         <Route path="/mysongs">
-          <p>Made it to mysongs</p>
+          <h1>My Songs</h1>
```

Comment:
Thanks for this change, I think this was failing in the tests bc the title was wrong